src/components/paradox/results/personality/career/inline-jobs/index.js

```diff
@@ -0,0 +1,138 @@
+import PropTypes from "prop-types";
+import {faCaretLeft, faCaretRight} from "@fortawesome/free-solid-svg-icons";
+import Icon from "lib/helpers/icon";
+import {useEffect, useState} from "react";
+import InlineJob from "./job";
+import style from "./style.scss";
+
+function CarouselButton({className, onClick, children, disabled}) {
+  const [isDisabled, setIsDisabled] = useState(disabled);
+
+  const handleClick = () => {
+    if(isDisabled) { return; }
+    onClick();
+  };
+
+  useEffect(() => {
+    setIsDisabled(disabled);
+  }, [disabled]);
+
+  return (
+    <div
+      className={`${className} ${isDisabled ? style.disabled : ""}`}
+      onClick={handleClick}
+      onKeyDown={handleClick}
+      role="button"
+      tabIndex={disabled ? -1 : 0}
+    >
+      {children}
+    </div>
+  );
+}
+CarouselButton.propTypes = {
+  children: PropTypes.node.isRequired,
+  className: PropTypes.string.isRequired,
+  disabled: PropTypes.bool.isRequired,
+  onClick: PropTypes.func.isRequired
+};
+
+function InlineJobs({className, count, jobs, jobSource, translate}) {
+  const [start, setStart] = useState(0);
+  const [end, setEnd] = useState(count || 0);
+
+  const jobSourceURL = {
+    Indeed: "https://www.indeed.com/",
+    Monster: "https://www.monster.com",
+    MyNextMove: "https://www.mynextmove.org"
+  };
+
+  const handlePrev = () => {
+    if(start - count <= 0) {
+      setStart(0);
+      setEnd(count);
+    } else {
+      setEnd(end - count);
+      setStart(start - count);
+    }
+  };
+  const handleNext = () => {
+    if(end + count >= jobs.length - 1) {
+      setStart(jobs.length - (count));
+      setEnd(jobs.length);
+    } else {
+      setStart(end);
+      setEnd(end + count);
+    }
+  };
+
+  useEffect(() => {
+    setStart(0);
+    setEnd(count);
+  }, [count]);
+
+  if(!(jobSource in jobSourceURL)) { return null; }
+
+  if(jobs?.length === 0) {
+    return (
+      <div className={`${style.job} ${className}`}>
+        <InlineJob
+          job={{}}
```

Comment:
and skip the job here

Reply:
This check is there to display "No Jobs Found" when an empty array is passed in.

Reply:
I was just saying you can leave out this specific line, so it passes job undefined

Reply:
Oh OK ! Job removed!